src/App.jsx

```diff
@@ -1,104 +1,33 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList/PostList';
+
+// Получение пользователей
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+// Получение комментов
+function getCommentById(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
```

Comment:
```suggestion
    commentsFromServer.filter(comment => comment.postId === postId)
```
no need to return null. filter will return an empty array - it's better and easier to handle this case. You don't need to check if it's null before calling some method or property like `post.comments.length`